ports/esp32/boards/WPI_PICO_W/board.json

```diff
@@ -0,0 +1,24 @@
+{
+    "deploy": [
+        "../deploy_s3.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "WiFi",
+        "External Flash",
+        "External RAM",
+        "USB-C"
+    ],
+    "images": [
+        "walnutpi_pico_w.jpg"
+    ],
+    "mcu": "esp32s3",
+    "product": "Pico W",
```

Comment:
This can't be called "Pico W" because there's already a Pico W board made by Raspberry Pi.

I'm not sure we want to support this board which is trying to be an ESP32 clone of the Pico W.  It's confusing for users to have it so similarly named as the RPi Pico W.